CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 ﻿cmake_minimum_required (VERSION 3.10)
-find_package(OpenMP)
+
+# Default to C++14 if not set so GTest/GMock can build
+if (NOT BLT_CXX_STD)
+    set(BLT_CXX_STD "c++14" CACHE STRING "")
+endif()
+
+include(external/blt/SetupBLT.cmake)
```

Comment:
What does BLT provide that this library need and would need be trivial to get from raw CMake?

Reply:
Mainly it has reasonable handling of options for OpenMP, cuda, hip and so-forth handled for normal .cpp files without having to continually rebuild all of that from scratch. For camp I’ve admittedly avoided it so far, but it’s a quick way to bootstrap what we need for a multi-backend project and also includes gtest as a built-in option. It doesn’t necessarily have to be a sub module, heck we don’t necessarily have to use it, but as a way to make progress relatively rapidly it made sense to me to start there.